homeassistant/components/feedreader.py

```diff
@@ -0,0 +1,93 @@
+"""RSS/Atom feed reader for Home Assistant."""
+from datetime import datetime
+from logging import getLogger
+from homeassistant.helpers import validate_config
+from homeassistant.helpers.event import track_utc_time_change
+
+
+REQUIREMENTS = ['feedparser==5.2.1']
+
+_LOGGER = getLogger(__name__)
+
+DOMAIN = "feedreader"
+EVENT_FEEDREADER = "feedreader"
+
+
+class FeedManager(object):
+    """Abstraction over feedparser module."""
+
+    def __init__(self, url, hass):
+        """Initialize the FeedManager object, poll every hour."""
+        self._url = url
+        self._feed = None
+        self._hass = hass
+        # Initialize last entry timestamp as epoch time
+        self._last_entry_timestamp = datetime.utcfromtimestamp(0).timetuple()
+        _LOGGER.debug('Loading feed %s', self._url)
+        track_utc_time_change(hass, lambda now: self._update(),
+                              minute=0, second=0)
+
+    @property
+    def url(self):
+        """Url of the feed."""
+        return self._url
+
+    @property
```

Comment:
Actually, I feel like this class can be as simple as having instance variables for `url`, `hass`, `feed`, `last_entry_timestamp` and an `update` method.